compose.yml

```diff
@@ -0,0 +1,8 @@
+services:
```

Comment:
We should add version
`version: "3.5"`

Reply:
in latest docker compose version (V2) this is not necessary. 

1. https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-optional 
2. https://docs.docker.com/compose/compose-file/
3. https://forums.docker.com/t/do-you-need-to-put-the-version-at-the-top-of-docker-compose-yml-file/135863